src/pages/Map.js

```diff
@@ -1,28 +1,34 @@
-import React, { useMemo, useState, useEffect, useCallback, useContext } from 'react';
-import { useSearchParams, useNavigate } from 'react-router-dom';
+import React, {
+  useMemo,
+  useState,
+  useEffect,
+  useCallback,
+  useContext,
+} from "react";
```

Comment:
Please don't replace all `'` with `"`

Reply:
My bad, I overlooked the changes from the lint. Edited already the file to present the changes that matters. Thanks